scm_staging/webhook.py

```diff
@@ -483,16 +483,24 @@ def make_app(app_config: AppConfig) -> Application:
     default=None,
     help="Filename for Tornado general log, default to console",
 )
+define(
+    "db_file",
+    default="submit_requests.db",
+    help="SQLite3 database tracking the submitrequests",
+)
 
 
 def main():
     from scm_staging.cleanup import process_all_stored_srs
+    from scm_staging import rabbit_listener as mq
 
     options.parse_command_line()
     LOGGER.configure_log_files(
         options.log, options.access_log, options.app_log, options.general_log
     )
     app_config = AppConfig.from_env()
+    mq.create_db(options.db_file)
+    mq.rabbit_listener(options.db_file)
```

Comment:
This doesn't work. The call to `rabbit_listener` blocks for ever and the tornado web application never launches.